api/core/storage/mongo/partials/mongo_organizations.py

```diff
@@ -404,6 +404,23 @@ async def feedback_slack_hook_for_tenant(self, tenant_uid: int) -> str | None:
             raise ObjectNotFoundException("Organization not found", code="organization_not_found")
         return doc.get("feedback_slack_hook")
 
+    @override
+    async def get_organization_by_slack_channel_id(self, slack_channel_id: str) -> TenantData | None:
+        cursor = self._collection.find({"slack_channel_id": slack_channel_id})
```

Comment:
We should index the field to avoid spamming the db. I'll do in a separate PR.

Reply:
You should just use `_get_public_org` ?

Reply:
-> https://github.com/WorkflowAI/WorkflowAI/pull/240

Reply:
thank you !